### PR TITLE
[main page] a full translation to Russian

### PR DESCRIPTION
**Pull request by iGusev**

### CODE REVIEW

File: app/views/index.volt [hunk 1]
```diff
@@ -26,28 +26,28 @@
...
+                            {{ tr('header_slogan') }}
                             <a id="bench-link" href="http://docs.phalconphp.com" target="_blank">
-                                See for yourself...
+                                {{ tr('see_for_youself') }}
                             </a>
                         </p>
 
                         <div class="home-button-container">
-                            {{ link_to(['for': 'pages', 'id': 'the-difference-tm', 'pageSlug': 'download', 'language': language], '<i class="fa fa-download"></i> Get Phalcon 2.0', 'class': 'btn btn-lg btn-phalcon') }}
+                            {{ link_to(['for': 'pages', 'id': 'the-difference-tm', 'pageSlug': 'download', 'language': language], '<i class="fa fa-download"></i> '~ tr('get_phalcon'), 'class': 'btn btn-lg btn-phalcon') }}
```
Comment (sergeyklay) [inline app/views/index.volt]: Better way
```volt
'~ tr('get_phalcon') ~ '2.0'
```

To avoid future i18n changes

Comment (iGusev) [inline app/views/index.volt]: done

**Comment by iGusev** [comment]
@sergeyklay @andresgutierrez ping!

**Comment by sergeyklay** [comment]
@andresgutierrez :+1:

**Comment by andresgutierrez** [comment]
Thanks

**Pull request merged**